controllers/operatorpolicy_controller.go

```diff
@@ -414,7 +514,7 @@ func (r *OperatorPolicyReconciler) handleSubscription(ctx context.Context, polic
 		// For now, just mark it as compliant
 		err := r.updateStatus(ctx, policy, matchesCond("Subscription"), matchedObj(foundSub))
 		if err != nil {
-			return fmt.Errorf("error updating the status for an OperatorGroup that matches: %w", err)
+			return fmt.Errorf("error updating the status for an Subscription that matches: %w", err)
```

Comment:
Good catch! Fix the `an` at the same time?
```suggestion
			return fmt.Errorf("error updating the status for a Subscription that matches: %w", err)
```

Reply:
Haha missed that!